src/solvers/smt2_incremental/smt2_incremental_decision_procedure.h

```diff
@@ -13,13 +13,14 @@
 #include <solvers/smt2_incremental/object_tracking.h>
 #include <solvers/smt2_incremental/smt_is_dynamic_object.h>
 #include <solvers/smt2_incremental/smt_object_size.h>
-#include <solvers/smt2_incremental/struct_encoding.h>
 #include <solvers/smt2_incremental/type_size_mapping.h>
 #include <solvers/stack_decision_procedure.h>
 
 #include <memory>
 #include <unordered_map>
 
+#include "solvers/smt2_incremental/encoding/struct_encoding.h"
```

Comment:
This probably needs to be `#include <solvers/smt2_incremental/encoding/struct_encoding.h>` to fix your build issues.

Reply:
Yep, changed